lxd/device/device_utils_disk.go

```diff
@@ -421,7 +425,10 @@ func DiskVMVirtiofsdStart(execPath string, inst instance.Instance, socketPath st
 	}
 
 	// Remove old socket if needed.
-	os.Remove(socketPath)
+	err := os.Remove(socketPath)
```

Comment:
I think we can actually get away with just ignoring the error here as the command will fail to start if it cant write its new socket file anyway.